ops/testing.py

```diff
@@ -2888,10 +2881,10 @@ def remove_path(self, path: str, *, recursive: bool = False):
             file_path.unlink()
 
     def _find_exec_handler(self, command: List[str]) -> Optional[ExecHandler]:
-        for command_prefix, handler in self._exec_handlers:
-            if tuple(command[:len(command_prefix)]) == command_prefix:
-                return handler
-        return None
+        for prefix_len in reversed(range(len(command) + 1)):
```

Comment:
Clever and clear approach to look it up this way -- I like it!

Reply:
Thanks!